sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -168,6 +173,21 @@ public AzureMonitorExporterBuilder connectionString(String connectionString) {
         if (endpoint != null) {
             this.endpoint(endpoint);
         }
+        String authorizationType = keyValues.get("Authorization");
+        if (authorizationType != null && authorizationType.equals("aad") && this.credential == null) {
```

Comment:
The spec calls for the string to be all capital.  I suggest you do equalsIgnoreCase(..) to be safe.